website/src/styles/theme.css

```
@@ -3,17 +3,32 @@
+ * The header is the exception, and is shared outright: header.css is generated
+ * from the app's own rules (see scripts/mirror-header.mjs) and loaded before
+ * this file, so the two sites cannot drift into different headers.
  *
  * Prose stays in Starlight's readable sans; JetBrains Mono is used for code,
- * the site title, and headings, matching the app's character without making
+ * the header, and headings, matching the app's character without making
  * long-form guides harder to read.
  */
 
 :root {
   --sl-font-mono: "JetBrains Mono", ui-monospace, SFMono-Regular, Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)